### ISSUE TITLE
Not the "most over power grabber on github"

### ISSUE THREAD

**Issue by vesperlol** [Mani175/Pirate-Cookie-Grabber#18]
- Not FUD once its compiled to EXE
- No exe version, only people with python and the requirements installed can run the app
- Complicated to setup (browser_cookie3)
- No protection
Not saying all this to hate ofc, its a very cool project, just use the right terms :P

**Comment by IgnHeaven** [comment]
- well tbh any python program once its compiled will give false positives.
- browser_cookie3 can be setup like any other library. you just need to upgrade pip first or it will give errors
- maybe its not the most overpowered but it gets the job done.

**Comment by vesperlol** [comment]
> * well tbh any python program once its compiled will give false positives.
> * browser_cookie3 can be setup like any other library. you just need to upgrade pip first or it will give errors
> * maybe its not the most overpowered but it gets the job done.

yes its python so its not overpowered lol..
and yeah you need c++ build tools for brower_cookie3, if you try to install it'll ask you to install c++ build tools microsft visual studio 2019

**Comment by IgnHeaven** [comment]
> 

yeah idk, i just had to upgrade my pip installer and i ran `pip install cookie_browser3` and done
